mon-pix/app/services/storage.js

```diff
@@ -0,0 +1,17 @@
+import Service from '@ember/service';
+
+const SESSIONSTORAGE_LOGIN = 'PIX_LOGIN';
+export default class Storage extends Service {
+  setLogin(login) {
+    sessionStorage.setItem(SESSIONSTORAGE_LOGIN, login);
+  }
+  getLogin() {
+    const login = sessionStorage.getItem(SESSIONSTORAGE_LOGIN);
+    if (login) {
+      sessionStorage.removeItem(SESSIONSTORAGE_LOGIN);
+      return login;
+    } else {
+      return '';
+    }
+  }
+}
```

Comment:
Ajouter des lignes entre certaines parties du code : 

```suggestion
import Service from '@ember/service';

const SESSIONSTORAGE_LOGIN = 'PIX_LOGIN';

export default class Storage extends Service {

  setLogin(login) {
    sessionStorage.setItem(SESSIONSTORAGE_LOGIN, login);
  }
  
  getLogin() {
    const login = sessionStorage.getItem(SESSIONSTORAGE_LOGIN);
    if (login) {
      sessionStorage.removeItem(SESSIONSTORAGE_LOGIN);
      return login;
    } else {
      return '';
    }
  }
}
```